examples/pydantic_models_to_grammar_examples.py

```diff
@@ -1,8 +1,14 @@
+#!/usr/bin/env python3
+
 # Function calling example using pydantic models.
```

Comment:
```suggestion
"""
Function calling example using pydantic models.
"""
```

Otherwise, `python3 examples/pydantic_models_to_grammar_examples.py --help` doesn't include the description. (which is taken from `sys.modules[__name__].__doc__` in `main()`)

Reply:
Oops, oversight, thanks for catching!